tests/testExport.py

```diff
@@ -152,19 +152,19 @@ def _testMultipage(self, extension, expectedMultipageCount):
                 os.remove(path)
 
     def test_multipage_png(self):
-        self._testMultipage(".png", 5)
+        self._testMultipage(".png", numFrames=5, expectedMultipageCount=5)
 
     def test_multipage_jpg(self):
-        self._testMultipage(".jpg", 5)
+        self._testMultipage(".jpg", numFrames=5, expectedMultipageCount=5)
 
     def test_multipage_svg(self):
-        self._testMultipage(".svg", 5)
+        self._testMultipage(".svg", numFrames=5, expectedMultipageCount=5)
 
     def test_multipage_gif(self):
-        self._testMultipage(".gif", 0)
+        self._testMultipage(".gif", numFrames=5, expectedMultipageCount=0)
 
     def test_multipage_pdf(self):
-        self._testMultipage(".pdf", 0)
+        self._testMultipage(".pdf", numFrames=5, expectedMultipageCount=0)
```

Comment:
What is being fixed in this commit?

Reply:
it only worked when the amount of frame was set to 5, any other number of frames would assert

Reply:
Did the tests fail???